mobile-controller/public/src/index.js

```diff
@@ -108,8 +109,8 @@ class App {
   }
 
   setupShooter() {
-
-    window.document.body.onclick = () => {
+    window.document.querySelector('html').onclick = () => {
```

Comment:
I changed it because as we rotate the body orientation to make the gun vertical, the click handler doesn't work the same. So to keep the click working in the entire page I got the body's parent element which is HTML. 

We can change the click handler back to body if we rotate a different element, like a `#root` container or so.

Reply:
got it!

Reply:
I changed it to allow shooting anywhere in the screen. With the handler on `body` it was only firing the gun when you click anywhere in the gun. Like this:

https://user-images.githubusercontent.com/41883467/203633585-798172bd-67c8-4075-b836-73dbdaa1d684.mov

This happens because when you have the click handler on body, the click is only perceived when the `propagation` is from an element inside the body. (which means if you click an element that is in the body, the click propagates and reaches the body, and it triggers your event handler.
However, your background is outside the body (on the parent element, which is html)
https://github.com/WellsSA/DuckHunt-JS/blob/d3d904916f1311d9470dc034be5a6ef9b33b7a73/mobile-controller/public/styles.css#L1-L2

So I changed the handler to html and the result is this:

https://user-images.githubusercontent.com/41883467/203633936-704f99e4-3d1a-44d8-9ed1-eecad2f73de3.mov